README.md

```diff
@@ -1,246 +1,211 @@
-|              | L7 authorization |                            |                            |                            | √ |
-|              | Cgroup-level isolation | √ |                            |                            |                            |
-| Traffic monitoring | Governance indicator monitoring |                            | √ |                            |                            |
-|              | End-to-End observability |                            |                            |                            | √ |
-| Programmable | Plug-in expansion capability |                            |                            |                            | √ |
-| Ecosystem collaboration | Data plane collaboration (Envoy etc.) |                            | √ |                            |                            |
-| Operating environment support | container                | √ |                            |                            |                            |
-
-## Contact
-
-If you have questions, feel free to reach out to us in the following ways:
-
-- [meeting notes](https://docs.google.com/document/d/1fFqolwWMVMk92yXPHvWGrMgsrb8Xru_v4Cve5ummjbk)
-- [mailing list](https://groups.google.com/forum/#!forum/kmesh)
-- [slack](https://cloud-native.slack.com/archives/C06BU2GB8NL)
-- [twitter](https://twitter.com/kmesh_net)
-
-## Contributing
-
-If you're interested in being a contributor and want to get involved in developing the Kmesh code, please see [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
-
-## License
-
-Kmesh is under the Apache 2.0 license. See the [LICENSE](LICENSE) file for details.
-
-Kmesh documentation is under the [CC-BY-4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode).
-
-## Credit
-
-This project was initially incubated in the [openEuler community](https://gitee.com/openeuler/Kmesh), thanks openEuler Community for the help on promoting this project in early days.
+<img src="docs/pics/logo/KMESH-horizontal-colour.png" alt="kmesh-logo" style="zoom: 100%;" />
+
+## Introduction
+
+Kmesh is a high-performance service mesh data plane software based on programmable kernel. Provides high-performance service communication infrastructure in service mesh scenarios.
+
+## Why Kmesh
+
+### Challenges of the Service Mesh Data Plane
+
+The service mesh software represented by Istio has gradually become popular and has become an important component of cloud infrastructure. However, the current service mesh still face some challenges:
+
+- **Extra latency overhead at the proxy layer**: Single hop service access increases by [2~3ms](https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#data-plane-performance), which cannot meet the SLA requirements of latency-sensitive applications. Although the community has come up with a variety of data plane solutions to this problem, the overhead introduced by agents cannot be completely reduced.
```

Comment:
if you take a look at the latest document, it's not that much.

Reply:
That was tested with the istio sidecar, both kmesh and istio tests are based on a same machine, so i think it is equal

Reply:
> Latency for Istio 1.21
Inside the mesh, a request traverses the client-side proxy and then the server-side proxy. In the default configuration of Istio 1.21.1 (i.e. Istio with telemetry v2), the two proxies add about 0.182 ms and 0.248 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency. We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/release-1.21/perf/benchmark) for the http/1.1 protocol, with a 1 kB payload at 1000 requests per second using 2,4,8,16,32,64 client connections, 2 proxy workers and mutual TLS enabled.

Reply:
I don't know there's so much improve in 1.21, but the what Kmesh said here(2-3ms) is not the latest.

Reply:
The test environment is same. @nlgwcy One point wanna to confirm, is the istio test with mtls and stats enabled